### DISCUSSION TITLE
Is there someway to end the session of an user after inactivity?

### DISCUSSION THREAD

**Discussion by RRNicholas** [requarks/wiki#4584] [Help / Questions]
I have tried it with a couple of browsers and there does not seem to be something like an logout after long idle. It would be very nice to have. Is there a way to implement it?

**Comment by NGPixel** [comment]
You can change the token expiration and renewal period values under Admin > Security. The defaults are 30 minutes and 14 days respectively.

**Reply by RRNicholas** [reply]
Thank you for replying. I have set the values to 10Minutes and 1Day.
It logs me out as the Cookie expires, but after it expires I have to delete the Cookie manually to log back in. Otherwise it's loading endlessly.
